### ISSUE TITLE
[space-in-parens] doesn't handle comments

### ISSUE THREAD

**Issue by Patrik-Lundqvist** [buzinas/tslint-eslint-rules#252]
```tslint: 5.7.0```
```typescript: 2.5.2```

**tslint.json:**
```json
"rules": {
    "space-in-parens": [true, "never"]
}
```

**Code:**
```ts
foo(/* test */ 'bar');
```

**Actual behavior:**
```sh
ERROR: there should be no spaces inside this paren.
```

**Expected  behavior:**
Should not throw an error

See: 
[eslint demo](https://eslint.org/demo/#eyJ0ZXh0IjoiY29uc3QgZm9vID0gKCkgPT4ge307XHJcbmZvbygvKiB0ZXN0ICovICdiYXInKTsiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjgsInNvdXJjZVR5cGUiOiJzY3JpcHQiLCJlY21hRmVhdHVyZXMiOnsianN4IjpmYWxzZSwiZ2xvYmFsUmV0dXJuIjpmYWxzZSwiaW1wbGllZFN0cmljdCI6dHJ1ZSwiZXhwZXJpbWVudGFsT2JqZWN0UmVzdFNwcmVhZCI6ZmFsc2V9fSwicnVsZXMiOnsiYWNjZXNzb3ItcGFpcnMiOiJvZmYiLCJhcnJheS1icmFja2V0LW5ld2xpbmUiOiJvZmYiLCJhcnJheS1icmFja2V0LXNwYWNpbmciOiJvZmYiLCJhcnJheS1jYWxsYmFjay1yZXR1cm4iOiJvZmYiLCJhcnJheS1lbGVtZW50LW5ld2xpbmUiOiJvZmYiLCJhcnJvdy1ib2R5LXN0eWxlIjoib2ZmIiwiYXJyb3ctcGFyZW5zIjoib2ZmIiwiYXJyb3ctc3BhY2luZyI6Im9mZiIsImJsb2NrLXNjb3BlZC12YXIiOiJvZmYiLCJibG9jay1zcGFjaW5nIjoib2ZmIiwiYnJhY2Utc3R5bGUiOiJvZmYiLCJjYWxsYmFjay1yZXR1cm4iOiJvZmYiLCJjYW1lbGNhc2UiOiJvZmYiLCJjYXBpdGFsaXplZC1jb21tZW50cyI6Im9mZiIsImNsYXNzLW1ldGhvZHMtdXNlLXRoaXMiOiJvZmYiLCJjb21tYS1kYW5nbGUiOiJvZmYiLCJjb21tYS1zcGFjaW5nIjoib2ZmIiwiY29tbWEtc3R5bGUiOiJvZmYiLCJjb21wbGV4aXR5Ijoib2ZmIiwiY29tcHV0ZWQtcHJvcGVydHktc3BhY2luZyI6Im9mZiIsImNvbnNpc3RlbnQtcmV0dXJuIjoib2ZmIiwiY29uc2lzdGVudC10aGlzIjoib2ZmIiwiY29uc3RydWN0b3Itc3VwZXIiOiJlcnJvciIsImN1cmx5Ijoib2ZmIiwiZGVmYXVsdC1jYXNlIjoib2ZmIiwiZG90LWxvY2F0aW9uIjoib2ZmIiwiZG90LW5vdGF0aW9uIjoib2ZmIiwiZW9sLWxhc3QiOiJvZmYiLCJlcWVxZXEiOiJvZmYiLCJmb3ItZGlyZWN0aW9uIjoib2ZmIiwiZnVuYy1jYWxsLXNwYWNpbmciOiJvZmYiLCJmdW5jLW5hbWUtbWF0Y2hpbmciOiJvZmYiLCJmdW5jLW5hbWVzIjoib2ZmIiwiZnVuYy1zdHlsZSI6Im9mZiIsImdlbmVyYXRvci1zdGFyLXNwYWNpbmciOiJvZmYiLCJnZXR0ZXItcmV0dXJuIjoib2ZmIiwiZ2xvYmFsLXJlcXVpcmUiOiJvZmYiLCJndWFyZC1mb3ItaW4iOiJvZmYiLCJoYW5kbGUtY2FsbGJhY2stZXJyIjoib2ZmIiwiaWQtYmxhY2tsaXN0Ijoib2ZmIiwiaWQtbGVuZ3RoIjoib2ZmIiwiaWQtbWF0Y2giOiJvZmYiLCJpbmRlbnQtbGVnYWN5Ijoib2ZmIiwiaW5kZW50Ijoib2ZmIiwiaW5pdC1kZWNsYXJhdGlvbnMiOiJvZmYiLCJqc3gtcXVvdGVzIjoib2ZmIiwia2V5LXNwYWNpbmciOiJvZmYiLCJrZXl3b3JkLXNwYWNpbmciOiJvZmYiLCJsaW5lLWNvbW1lbnQtcG9zaXRpb24iOiJvZmYiLCJsaW5lYnJlYWstc3R5bGUiOiJvZmYiLCJsaW5lcy1hcm91bmQtY29tbWVudCI6Im9mZiIsImxpbmVzLWFyb3VuZC1kaXJlY3RpdmUiOiJvZmYiLCJtYXgtZGVwdGgiOiJvZmYiLCJtYXgtbGVuIjoib2ZmIiwibWF4LWxpbmVzIjoib2ZmIiwibWF4LW5lc3RlZC1jYWxsYmFja3MiOiJvZmYiLCJtYXgtcGFyYW1zIjoib2ZmIiwibWF4LXN0YXRlbWVudHMtcGVyLWxpbmUiOiJvZmYiLCJtYXgtc3RhdGVtZW50cyI6Im9mZiIsIm11bHRpbGluZS10ZXJuYXJ5Ijoib2ZmIiwibmV3LWNhcCI6Im9mZiIsIm5ldy1wYXJlbnMiOiJvZmYiLCJuZXdsaW5lLWFmdGVyLXZhciI6Im9mZiIsIm5ld2xpbmUtYmVmb3JlLXJldHVybiI6Im9mZiIsIm5ld2xpbmUtcGVyLWNoYWluZWQtY2FsbCI6Im9mZiIsIm5vLWFsZXJ0Ijoib2ZmIiwibm8tYXJyYXktY29uc3RydWN0b3IiOiJvZmYiLCJuby1hd2FpdC1pbi1sb29wIjoib2ZmIiwibm8tYml0d2lzZSI6Im9mZiIsIm5vLWJ1ZmZlci1jb25zdHJ1Y3RvciI6Im9mZiIsIm5vLWNhbGxlciI6Im9mZiIsIm5vLWNhc2UtZGVjbGFyYXRpb25zIjoiZXJyb3IiLCJuby1jYXRjaC1zaGFkb3ciOiJvZmYiLCJuby1jbGFzcy1hc3NpZ24iOiJlcnJvciIsIm5vLWNvbXBhcmUtbmVnLXplcm8iOiJlcnJvciIsIm5vLWNvbmQtYXNzaWduIjoiZXJyb3IiLCJuby1jb25mdXNpbmctYXJyb3ciOiJvZmYiLCJuby1jb25zb2xlIjoiZXJyb3IiLCJuby1jb25zdC1hc3NpZ24iOiJlcnJvciIsIm5vLWNvbnN0YW50LWNvbmRpdGlvbiI6ImVycm9yIiwibm8tY29udGludWUiOiJvZmYiLCJuby1jb250cm9sLXJlZ2V4IjoiZXJyb3IiLCJuby1kZWJ1Z2dlciI6ImVycm9yIiwibm8tZGVsZXRlLXZhciI6ImVycm9yIiwibm8tZGl2LXJlZ2V4Ijoib2ZmIiwibm8tZHVwZS1hcmdzIjoiZXJyb3IiLCJuby1kdXBlLWNsYXNzLW1lbWJlcnMiOiJlcnJvciIsIm5vLWR1cGUta2V5cyI6ImVycm9yIiwibm8tZHVwbGljYXRlLWNhc2UiOiJlcnJvciIsIm5vLWR1cGxpY2F0ZS1pbXBvcnRzIjoib2ZmIiwibm8tZWxzZS1yZXR1cm4iOiJvZmYiLCJuby1lbXB0eS1jaGFyYWN0ZXItY2xhc3MiOiJlcnJvciIsIm5vLWVtcHR5LWZ1bmN0aW9uIjoib2ZmIiwibm8tZW1wdHktcGF0dGVybiI6ImVycm9yIiwibm8tZW1wdHkiOiJlcnJvciIsIm5vLWVxLW51bGwiOiJvZmYiLCJuby1ldmFsIjoib2ZmIiwibm8tZXgtYXNzaWduIjoiZXJyb3IiLCJuby1leHRlbmQtbmF0aXZlIjoib2ZmIiwibm8tZXh0cmEtYmluZCI6Im9mZiIsIm5vLWV4dHJhLWJvb2xlYW4tY2FzdCI6ImVycm9yIiwibm8tZXh0cmEtbGFiZWwiOiJvZmYiLCJuby1leHRyYS1wYXJlbnMiOiJvZmYiLCJuby1leHRyYS1zZW1pIjoiZXJyb3IiLCJuby1mYWxsdGhyb3VnaCI6ImVycm9yIiwibm8tZmxvYXRpbmctZGVjaW1hbCI6Im9mZiIsIm5vLWZ1bmMtYXNzaWduIjoiZXJyb3IiLCJuby1nbG9iYWwtYXNzaWduIjoiZXJyb3IiLCJuby1pbXBsaWNpdC1jb2VyY2lvbiI6Im9mZiIsIm5vLWltcGxpY2l0LWdsb2JhbHMiOiJvZmYiLCJuby1pbXBsaWVkLWV2YWwiOiJvZmYiLCJuby1pbmxpbmUtY29tbWVudHMiOiJvZmYiLCJuby1pbm5lci1kZWNsYXJhdGlvbnMiOiJlcnJvciIsIm5vLWludmFsaWQtcmVnZXhwIjoiZXJyb3IiLCJuby1pbnZhbGlkLXRoaXMiOiJvZmYiLCJuby1pcnJlZ3VsYXItd2hpdGVzcGFjZSI6ImVycm9yIiwibm8taXRlcmF0b3IiOiJvZmYiLCJuby1sYWJlbC12YXIiOiJvZmYiLCJuby1sYWJlbHMiOiJvZmYiLCJuby1sb25lLWJsb2NrcyI6Im9mZiIsIm5vLWxvbmVseS1pZiI6Im9mZiIsIm5vLWxvb3AtZnVuYyI6Im9mZiIsIm5vLW1hZ2ljLW51bWJlcnMiOiJvZmYiLCJuby1taXhlZC1vcGVyYXRvcnMiOiJvZmYiLCJuby1taXhlZC1yZXF1aXJlcyI6Im9mZiIsIm5vLW1peGVkLXNwYWNlcy1hbmQtdGFicyI6ImVycm9yIiwibm8tbXVsdGktYXNzaWduIjoib2ZmIiwibm8tbXVsdGktc3BhY2VzIjoib2ZmIiwibm8tbXVsdGktc3RyIjoib2ZmIiwibm8tbXVsdGlwbGUtZW1wdHktbGluZXMiOiJvZmYiLCJuby1uYXRpdmUtcmVhc3NpZ24iOiJvZmYiLCJuby1uZWdhdGVkLWNvbmRpdGlvbiI6Im9mZiIsIm5vLW5lZ2F0ZWQtaW4tbGhzIjoib2ZmIiwibm8tbmVzdGVkLXRlcm5hcnkiOiJvZmYiLCJuby1uZXctZnVuYyI6Im9mZiIsIm5vLW5ldy1vYmplY3QiOiJvZmYiLCJuby1uZXctcmVxdWlyZSI6Im9mZiIsIm5vLW5ldy1zeW1ib2wiOiJlcnJvciIsIm5vLW5ldy13cmFwcGVycyI6Im9mZiIsIm5vLW5ldyI6Im9mZiIsIm5vLW9iai1jYWxscyI6ImVycm9yIiwibm8tb2N0YWwtZXNjYXBlIjoib2ZmIiwibm8tb2N0YWwiOiJlcnJvciIsIm5vLXBhcmFtLXJlYXNzaWduIjoib2ZmIiwibm8tcGF0aC1jb25jYXQiOiJvZmYiLCJuby1wbHVzcGx1cyI6Im9mZiIsIm5vLXByb2Nlc3MtZW52Ijoib2ZmIiwibm8tcHJvY2Vzcy1leGl0Ijoib2ZmIiwibm8tcHJvdG8iOiJvZmYiLCJuby1wcm90b3R5cGUtYnVpbHRpbnMiOiJvZmYiLCJuby1yZWRlY2xhcmUiOiJlcnJvciIsIm5vLXJlZ2V4LXNwYWNlcyI6ImVycm9yIiwibm8tcmVzdHJpY3RlZC1nbG9iYWxzIjoib2ZmIiwibm8tcmVzdHJpY3RlZC1pbXBvcnRzIjoib2ZmIiwibm8tcmVzdHJpY3RlZC1tb2R1bGVzIjoib2ZmIiwibm8tcmVzdHJpY3RlZC1wcm9wZXJ0aWVzIjoib2ZmIiwibm8tcmVzdHJpY3RlZC1zeW50YXgiOiJvZmYiLCJuby1yZXR1cm4tYXNzaWduIjoib2ZmIiwibm8tcmV0dXJuLWF3YWl0Ijoib2ZmIiwibm8tc2NyaXB0LXVybCI6Im9mZiIsIm5vLXNlbGYtYXNzaWduIjoiZXJyb3IiLCJuby1zZWxmLWNvbXBhcmUiOiJvZmYiLCJuby1zZXF1ZW5jZXMiOiJvZmYiLCJuby1zaGFkb3ctcmVzdHJpY3RlZC1uYW1lcyI6Im9mZiIsIm5vLXNoYWRvdyI6Im9mZiIsIm5vLXNwYWNlZC1mdW5jIjoib2ZmIiwibm8tc3BhcnNlLWFycmF5cyI6ImVycm9yIiwibm8tc3luYyI6Im9mZiIsIm5vLXRhYnMiOiJvZmYiLCJuby10ZW1wbGF0ZS1jdXJseS1pbi1zdHJpbmciOiJvZmYiLCJuby10ZXJuYXJ5Ijoib2ZmIiwibm8tdGhpcy1iZWZvcmUtc3VwZXIiOiJlcnJvciIsIm5vLXRocm93LWxpdGVyYWwiOiJvZmYiLCJuby10cmFpbGluZy1zcGFjZXMiOiJvZmYiLCJuby11bmRlZi1pbml0Ijoib2ZmIiwibm8tdW5kZWYiOiJlcnJvciIsIm5vLXVuZGVmaW5lZCI6Im9mZiIsIm5vLXVuZGVyc2NvcmUtZGFuZ2xlIjoib2ZmIiwibm8tdW5leHBlY3RlZC1tdWx0aWxpbmUiOiJlcnJvciIsIm5vLXVubW9kaWZpZWQtbG9vcC1jb25kaXRpb24iOiJvZmYiLCJuby11bm5lZWRlZC10ZXJuYXJ5Ijoib2ZmIiwibm8tdW5yZWFjaGFibGUiOiJlcnJvciIsIm5vLXVuc2FmZS1maW5hbGx5IjoiZXJyb3IiLCJuby11bnNhZmUtbmVnYXRpb24iOiJlcnJvciIsIm5vLXVudXNlZC1leHByZXNzaW9ucyI6Im9mZiIsIm5vLXVudXNlZC1sYWJlbHMiOiJlcnJvciIsIm5vLXVudXNlZC12YXJzIjoib2ZmIiwibm8tdXNlLWJlZm9yZS1kZWZpbmUiOiJlcnJvciIsIm5vLXVzZWxlc3MtY2FsbCI6Im9mZiIsIm5vLXVzZWxlc3MtY29tcHV0ZWQta2V5Ijoib2ZmIiwibm8tdXNlbGVzcy1jb25jYXQiOiJvZmYiLCJuby11c2VsZXNzLWNvbnN0cnVjdG9yIjoib2ZmIiwibm8tdXNlbGVzcy1lc2NhcGUiOiJlcnJvciIsIm5vLXVzZWxlc3MtcmVuYW1lIjoib2ZmIiwibm8tdXNlbGVzcy1yZXR1cm4iOiJvZmYiLCJuby12YXIiOiJvZmYiLCJuby12b2lkIjoib2ZmIiwibm8td2FybmluZy1jb21tZW50cyI6Im9mZiIsIm5vLXdoaXRlc3BhY2UtYmVmb3JlLXByb3BlcnR5Ijoib2ZmIiwibm8td2l0aCI6Im9mZiIsIm5vbmJsb2NrLXN0YXRlbWVudC1ib2R5LXBvc2l0aW9uIjoib2ZmIiwib2JqZWN0LWN1cmx5LW5ld2xpbmUiOiJvZmYiLCJvYmplY3QtY3VybHktc3BhY2luZyI6Im9mZiIsIm9iamVjdC1wcm9wZXJ0eS1uZXdsaW5lIjoib2ZmIiwib2JqZWN0LXNob3J0aGFuZCI6Im9mZiIsIm9uZS12YXItZGVjbGFyYXRpb24tcGVyLWxpbmUiOiJvZmYiLCJvbmUtdmFyIjoib2ZmIiwib3BlcmF0b3ItYXNzaWdubWVudCI6Im9mZiIsIm9wZXJhdG9yLWxpbmVicmVhayI6Im9mZiIsInBhZGRlZC1ibG9ja3MiOiJvZmYiLCJwYWRkaW5nLWxpbmUtYmV0d2Vlbi1zdGF0ZW1lbnRzIjoib2ZmIiwicHJlZmVyLWFycm93LWNhbGxiYWNrIjoib2ZmIiwicHJlZmVyLWNvbnN0Ijoib2ZmIiwicHJlZmVyLWRlc3RydWN0dXJpbmciOiJvZmYiLCJwcmVmZXItbnVtZXJpYy1saXRlcmFscyI6Im9mZiIsInByZWZlci1wcm9taXNlLXJlamVjdC1lcnJvcnMiOiJvZmYiLCJwcmVmZXItcmVmbGVjdCI6Im9mZiIsInByZWZlci1yZXN0LXBhcmFtcyI6Im9mZiIsInByZWZlci1zcHJlYWQiOiJvZmYiLCJwcmVmZXItdGVtcGxhdGUiOiJvZmYiLCJxdW90ZS1wcm9wcyI6Im9mZiIsInF1b3RlcyI6Im9mZiIsInJhZGl4Ijoib2ZmIiwicmVxdWlyZS1hd2FpdCI6Im9mZiIsInJlcXVpcmUtanNkb2MiOiJvZmYiLCJyZXF1aXJlLXlpZWxkIjoiZXJyb3IiLCJyZXN0LXNwcmVhZC1zcGFjaW5nIjoib2ZmIiwic2VtaS1zcGFjaW5nIjoib2ZmIiwic2VtaS1zdHlsZSI6Im9mZiIsInNlbWkiOiJvZmYiLCJzb3J0LWltcG9ydHMiOiJvZmYiLCJzb3J0LWtleXMiOiJvZmYiLCJzb3J0LXZhcnMiOiJvZmYiLCJzcGFjZS1iZWZvcmUtYmxvY2tzIjoib2ZmIiwic3BhY2UtYmVmb3JlLWZ1bmN0aW9uLXBhcmVuIjoib2ZmIiwic3BhY2UtaW4tcGFyZW5zIjoyLCJzcGFjZS1pbmZpeC1vcHMiOiJvZmYiLCJzcGFjZS11bmFyeS1vcHMiOiJvZmYiLCJzcGFjZWQtY29tbWVudCI6Im9mZiIsInN0cmljdCI6Im9mZiIsInN3aXRjaC1jb2xvbi1zcGFjaW5nIjoib2ZmIiwic3ltYm9sLWRlc2NyaXB0aW9uIjoib2ZmIiwidGVtcGxhdGUtY3VybHktc3BhY2luZyI6Im9mZiIsInRlbXBsYXRlLXRhZy1zcGFjaW5nIjoib2ZmIiwidW5pY29kZS1ib20iOiJvZmYiLCJ1c2UtaXNuYW4iOiJlcnJvciIsInZhbGlkLWpzZG9jIjoib2ZmIiwidmFsaWQtdHlwZW9mIjoiZXJyb3IiLCJ2YXJzLW9uLXRvcCI6Im9mZiIsIndyYXAtaWlmZSI6Im9mZiIsIndyYXAtcmVnZXgiOiJvZmYiLCJ5aWVsZC1zdGFyLXNwYWNpbmciOiJvZmYiLCJ5b2RhIjoib2ZmIn0sImVudiI6eyJicm93c2VyIjpmYWxzZSwibm9kZSI6ZmFsc2UsImNvbW1vbmpzIjpmYWxzZSwic2hhcmVkLW5vZGUtYnJvd3NlciI6ZmFsc2UsIndvcmtlciI6ZmFsc2UsImFtZCI6ZmFsc2UsIm1vY2hhIjpmYWxzZSwiamFzbWluZSI6ZmFsc2UsImplc3QiOmZhbHNlLCJwaGFudG9tanMiOmZhbHNlLCJqcXVlcnkiOmZhbHNlLCJxdW5pdCI6ZmFsc2UsInByb3RvdHlwZWpzIjpmYWxzZSwic2hlbGxqcyI6ZmFsc2UsIm1ldGVvciI6ZmFsc2UsIm1vbmdvIjpmYWxzZSwicHJvdHJhY3RvciI6ZmFsc2UsImFwcGxlc2NyaXB0IjpmYWxzZSwibmFzaG9ybiI6ZmFsc2UsInNlcnZpY2V3b3JrZXIiOmZhbHNlLCJhdG9tdGVzdCI6ZmFsc2UsImVtYmVydGVzdCI6ZmFsc2UsIndlYmV4dGVuc2lvbnMiOmZhbHNlLCJlczYiOmZhbHNlLCJncmVhc2Vtb25rZXkiOmZhbHNlfX19)

[eslint test](https://github.com/eslint/eslint/blob/61f109343503c13de7b742669e5df44964c9e4e7/tests/lib/rules/space-in-parens.js#L52)

**Comment by buzinas** [comment]
Hey @Patrik-Lundqvist, thanks for logging this bug report.

Please, feel free to open a PR for fixing this, if you'd like to :)

**Comment by d-fischer** [comment]
This bug breaks Webpack's named import feature, *even if there is no space between the comment and the path*.

```js
import(/* webpackChunkName: "dashboard-chunk" */'./views/Dashboard')
```
